src/d_pwr_led.py

```diff
@@ -0,0 +1,92 @@
+import os
+import time
+
+
+
+import time
+from time import sleep
+from threading import Event
+from led import CLed
+from button import CButton
+from led_pwr_callback import *
+import RPi.GPIO as GPIO
+
+# Globals
+##############################################################################
+# power led
+PWR_LED = CLed(led_pwr_cbck_led_init,
+               led_pwr_cbck_on,
+               led_pwr_cbck_off,
+               led_pwr_cbck_state)
+
+# button on gpio 22
+BUTTON_CHANNEL = 22
+BUTTON  = CButton(BUTTON_CHANNEL)
+# duty time within the range in [s] the duty time in ms is valid
+DUTY_TIME = {
+    range(0,1): 500,
+    range(1,5): 250,
+    range(5,10): 50,
+}
+
+def my_callback(channel):
+    """callback function for gpio event
+
+    Args:
+        channel: pressed button input pin
+    """
+    start = time.perf_counter()
+    duty_time_ms = None
+    duty_time_ms_tmp = None
+
+    while BUTTON.status(): # and check again the input
+        #print (BUTTON.status())
+        sleep(0.5)  # confirm the movement by waiting 1.5 sec
```

Comment:
# confirm the movement by waiting 0.5 sec

Reply:
habe ich ausprobiert und ist ersteinmal gut so